test/babelish/test_android2csv.rb

```diff
@@ -28,6 +28,26 @@ def test_initialize
       assert_equal filenames, converter.filenames
     end
 
+  def test_special_chars
+    csv_filename = "./android_special_chars_output.csv"
+    filenames = "test/data/android_special_chars.xml"
+    headers = %w{"constants german"}
+
+    converter = Babelish::Android2CSV.new({
+      :csv_filename => csv_filename,
+      :headers => %w{variables german},
+      :filenames => [filenames]
+    })
+
+    converter.convert
+
+    assert File.exist?(converter.csv_filename)
+    assert FileUtils.compare_file(csv_filename, 'test/data/android_special_chars.csv')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.